dipy/segment/tissue.py

```diff
@@ -3,7 +3,7 @@
 from dipy.segment.mrf import ConstantObservationModel, IteratedConditionalModes
 from dipy.sims.voxel import add_noise
 from dipy.testing.decorators import warning_for_keywords
-
+from sklearn.linear_model import LinearRegression
```

Comment:
scikit-learn is an optional package. Please, look at the codebase to get some inspiration of `optional_package` use.

Reply:
Added optional_package import